src/stub/parser.rs

```diff
@@ -298,6 +284,13 @@ impl<'a> Parser<'a> {
         self.token_stream.next()
     }
 
+    fn rest_of_line(&mut self) -> Option<String> {
+        match self.tokens_upto_newline() {
+            Some(tokens) => Some(tokens.join(" ").trim().to_string()),
+            None => None,
+        }
```

Comment:
This could have been written:

```
    fn rest_of_line(&mut self) -> Option<String> {
        Some(self.tokens_upto_newline()?.join(" ").trim().to_string())
    }
```

Reply:
Thanks, updated.